src/bun.js/bindings/webcore/EventEmitter.cpp

```diff
@@ -234,7 +234,7 @@ void EventEmitter::innerInvokeEventListeners(const Identifier& eventType, Simple
             continue;
 
         WTF::NakedPtr<JSC::Exception> exceptionPtr;
-        JSC::call(lexicalGlobalObject, jsFunction, callData, thisValue, arguments, exceptionPtr);
+        call(lexicalGlobalObject, jsFunction, callData, thisValue, arguments, exceptionPtr);
```

Comment:
@paperdave was it intentional we weren't doing async context tracking in EventEmitter?